packages/react/src/components/Masthead/MastheadSearch.js

```diff
@@ -292,24 +356,49 @@ const MastheadSearch = ({ placeHolderText, renderValue, searchOpenOnload }) => {
   return (
     <div
       data-autoid={`${stablePrefix}--masthead__search`}
-      className={className}>
-      <form action={_redirectUrl} method="get">
-        <input type="hidden" name="lang" value={state.lc} />
-        <input type="hidden" name="cc" value={state.cc} />
-        <input type="hidden" name="lnk" value="mhsrch" />
-        <Autosuggest
-          suggestions={state.suggestions} // The state value of suggestion
-          onSuggestionsFetchRequested={onSuggestionsFetchRequest} // Method to fetch data (should be async call)
-          onSuggestionsClearRequested={onSuggestionsClearedRequested} // When input bar loses focus
-          getSuggestionValue={_getSuggestionValue} // Name of suggestion
-          renderSuggestion={renderSuggestion} // How to display a suggestion
-          onSuggestionSelected={onSuggestionSelected} // When a suggestion is selected
-          highlightFirstSuggestion // First suggestion is highlighted by default
-          inputProps={inputProps}
-          renderInputComponent={renderInputComponent}
-          shouldRenderSuggestions={shouldRenderSuggestions}
-        />
-      </form>
+      className={className}
+      ref={ref}>
+      {state.isSearchOpen && (
+        <form
+          id={`${prefix}--masthead__search--form`}
+          action={_redirectUrl}
+          method="get">
+          <input type="hidden" name="lang" value={state.lc} />
+          <input type="hidden" name="cc" value={state.cc} />
+          <input type="hidden" name="lnk" value="mhsrch" />
+          <Autosuggest
+            suggestions={state.suggestions} // The state value of suggestion
+            onSuggestionsFetchRequested={onSuggestionsFetchRequest} // Method to fetch data (should be async call)
+            onSuggestionsClearRequested={onSuggestionsClearedRequested} // When input bar loses focus
+            getSuggestionValue={_getSuggestionValue} // Name of suggestion
+            renderSuggestion={renderSuggestion} // How to display a suggestion
+            onSuggestionSelected={onSuggestionSelected} // When a suggestion is selected
+            highlightFirstSuggestion // First suggestion is highlighted by default
+            inputProps={inputProps}
+            renderInputComponent={renderInputComponent}
+            shouldRenderSuggestions={shouldRenderSuggestions}
+          />
+        </form>
+      )}
+      <div className={`${prefix}--header__search--actions`}>
+        <HeaderGlobalAction
+          onClick={searchIconClick}
+          aria-label={
+            state.isSearchOpen ? 'Search all of IBM' : 'Open IBM search field'
```

Comment:
@kennylam do we have these label fields at all?

Reply:
@jeffchew We don't. I made a comment in the PR description about needing translations for these. We have no `search` section in our translation file right now. I'm going to create one for this and probably do a draft PR to make sure we're covering everything.

Reply:
ok groovy